packages/plugin-requirejs/src/processing.ts

```diff
@@ -107,13 +107,14 @@ export function relativeToFile(name: string, file: string): string {
 }
 
 interface Require {
-  nodeRequire(name: string): any;
+  nodeRequire(name: string): unknown;
 }
 
+// tslint:disable-next-line:no-reserved-keywords
 declare const require: Require;
 
-export function loadFromFile(url: string, callback: Function, errback: Function) {
-  const fs = require.nodeRequire('fs');
+export function loadFromFile(url: string, callback: (content: string) => void, errback: (error: Error) => void): void {
+  const fs = require.nodeRequire('fs') as {readFileSync(url: string, encodig: string): string};
```

Comment:
To fix a small typo and make it just a little bit more accurate in the subset of `readFileSync` that it represents
```suggestion
  const fs = require.nodeRequire('fs') as {readFileSync(path: string, options?: { encoding?: string } | string): string};
```

Reply:
Done